src/presenters/admin/alert-presenter.php

```diff
@@ -0,0 +1,104 @@
+<?php
+/**
+ * Presenter class for Alert boxes.
+ *
+ * @package Yoast\YoastSEO\Presenters\Admin
+ */
+
+namespace Yoast\WP\SEO\Presenters\Admin;
+
+use Yoast\WP\SEO\Presenters\Abstract_Presenter;
+
+/**
+ * Represents the class for Alerts.
+ */
+class Alert_Presenter extends Abstract_Presenter {
+
+	/**
+	 * Alert type.
+	 *
+	 * @var string
+	 */
+	const ERROR = 'error';
+
+	/**
+	 * Alert type.
+	 *
+	 * @var string
+	 */
+	const INFO = 'info';
+
+	/**
+	 * Alert type.
+	 *
+	 * @var string
+	 */
+	const SUCCESS = 'success';
+
+	/**
+	 * Alert type.
+	 *
+	 * @var string
+	 */
+	const WARNING = 'warning';
```

Comment:
As far as I'm concerned, none of this is needed. We can just make it so you instantiate the class with one of these 4 types without needing these constants.